src/libs/actions/Session/index.js

```diff
@@ -289,6 +289,46 @@ function resetPassword() {
     API.write('RequestPasswordReset', {
         email: credentials.login,
     },
+    {
+        optimisticData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    isLoading: true,
+                    forgotPassword: true,
+                    message: null,
+                    errors: null,
+                },
+            },
+        ],
+        successData: [
+            {
+                onyxMethod: CONST.ONYX.METHOD.MERGE,
+                key: ONYXKEYS.ACCOUNT,
+                value: {
+                    isLoading: false,
+                    message: null,
```

Comment:
NAB I'm wondering if we actually need to set `message: null` here. We already clear `message` in `optimisticData` so it should already be empty. This applies for `failureData` and for `resendResetPassword` too.

Reply:
Yeah my only thought was if something else would have set this `message` key previously?

Though I just tested this and don't think it's possible, here's what I did
1. sign up with a new account, click resend to resend a 2nd validation link and get the success message which also uses `account.message`.
2. validate the account outside of going through new dot (I simulated this with cli tools to validate for me)
3. go back to "start" page and enter email again and then click `forgot?`. Here is where I was expecting the `message` previous to persist but it didn't so I think we're okay. There is probably a corner case somewhere lurking if we don't probably clear out `account.message` but we can fix that if it ever shows up easily.

Reply:
Whenever we tap `Forgot?` we trigger [resetPassword](https://github.com/Expensify/App/blob/b9e3ee8f76fcc0b69aae5c30dc8ba24ff9e00452/src/pages/signin/PasswordForm.js#L154) and we clear `message` and `error` in [optimisticData](https://github.com/Expensify/App/blob/b9e3ee8f76fcc0b69aae5c30dc8ba24ff9e00452/src/libs/actions/Session/index.js#L300-L301), so I think that we should be ok.